backend/src/main/java/dccs/academy/dtos/CertificateDto.java

```diff
@@ -13,7 +13,7 @@ public class CertificateDto {
     private LocalDate validTo;
     private CertificateType certificateType;
     private String pdfUrl;
-    private SupplierEntity supplier;
+    private SupplierDto supplier;
     private List<UserDto> certificateAssignedUsers;
```

Comment:
In general, if you already have 'Certificate' in name, you don't need to repeat it in field, so assignedUsers is sufficient, as well as type instead of certificateType.